frontend/src/components/SortableTaskList.module.scss

```diff
@@ -34,3 +34,20 @@
   position: sticky;
   top: 0;
 }
+
+.taskName {
```

Comment:
This and `.measureTaskName` are same as in TaskMap.module.scss, apart from the additional dragging styles.
Maybe these could be defined at one place